ynr/apps/candidates/models/popolo_extra.py

```diff
@@ -289,6 +301,43 @@ def has_results(self):
             return True
         return False
 
+    @property
+    def uncontested(self):
+        if not self.candidates_locked:
+            return False
+        if self.get_winner_count == self.membership_set.count():
+            return True
+        return False
+
+    def mark_uncontested_winners(self, request, ip_address, log=True):
+        """
+        If the election is uncontested mark all candidates as elected
+        """
+        if not self.uncontested:
+            return
+
+        self.membership_set.update(elected=True)
+
+        if not log:
+            return
+        LoggedAction.objects.create(
+            user=request.user,
+            action_type="constituency-{}".format("-lock"),
```

Comment:
I think this (and the below) formats to `constituency--lock` (double hyphen). I've seen this show up in the data, so maybe there's a bug somewhere else that causes this same formatting too.

It should be `constituency-lock`, so I don't think the `format()` is adding anything here?